R/use_extendr.R

```diff
@@ -16,13 +16,15 @@
 #' If `NULL`, sanitized R package name is used instead.
 #' @param quiet Logical indicating whether any progress messages should be
 #'   generated or not. Also checks the `usethis.quiet` option.
+#' @param edition String indicating which Rust edition is used; Default `"2021"`.
 #' @return A logical value (invisible) indicating whether any package files were
 #' generated or not.
 #' @export
 use_extendr <- function(path = ".",
                         crate_name = NULL,
                         lib_name = NULL,
-                        quiet = getOption("usethis.quiet", FALSE)) {
+                        quiet = getOption("usethis.quiet", FALSE),
+                        edition = c("2021", "2018", "2015")) {
```

Comment:
Does it work with `2015` edition? `extendr` has `2018` edition and it seems you cannot reference `2018` package from `2015` project, can you?

Reply:
Let us remove `2015` edition. When compiling a package targeting `2015`, I get the following: 
![image](https://user-images.githubusercontent.com/8782986/168021949-74bae146-c11f-460d-a74c-5520bcf27bd1.png)